internal/logging/logging.go

```diff
@@ -0,0 +1,185 @@
+package logging
+
+import (
+	"fmt"
+	"go.uber.org/atomic"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+	"os"
+	"sync"
+	"time"
+)
+
+// Logging implements logging using: a switch to disable logging on Fatal log, default logger and level,
```

Comment:
Logging implements access to a default logger and named child loggers.
Log levels can be configured per child via Options.
Logs either to the console or the systemd journal.
Provides Fatal log to stop all loggers before the error is logged and the program exits.